File: Move Zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int size = nums.size();
        int anchor,temp;
        anchor = 0;
        for(int exp = 0;exp<size;exp++){
            if(nums[exp] != 0){
                temp = nums[exp];
                nums[exp] = nums[anchor];
                nums[anchor] = temp;
                anchor++;
            }
        }
    }
};

File: Move Zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        size = len(nums)
        moves = 0
        for i in range(size):
            if nums[i] == 0:
                moves+=1
            else:
                temp = nums[i]
                nums[i] = nums[i-moves]
                nums[i-moves] = temp
                   
                        
        
                
                
        
        
        
